droid-binary/bin/droid.sh

```diff
@@ -146,7 +146,7 @@ fi
 
 # Run the command-line or user interface version with the options:
 if [ $# -gt 0 ]; then
-    java -XX:+IgnoreUnrecognizedVMOptions --add-modules=java.xml.bind,java.xml.ws,java.xml.ws.annotation $OPTIONS -jar "$DROID_HOME/droid-command-line-${project.version}.jar" "$@"
+    java -XX:+IgnoreUnrecognizedVMOptions $OPTIONS -jar "$DROID_HOME/droid-command-line-${project.version}.jar" "$@"
```

Comment:
`-XX:+IgnoreUnrecognizedVMOptions` is only needed to allow `--add-modules` on JDKs prior to 9.

As you have removed the `--add-modules` argument you can also drop the `-XX:+IgnoreUnrecognizedVMOptions` argument.